framework/auth/views.py

```diff
@@ -873,6 +872,7 @@ def send_confirm_email(user, email, renew=False, external_id_provider=None, exte
             'osf_support_email': settings.OSF_SUPPORT_EMAIL,
             **merge_account_data,
         },
+        save=False
```

Comment:
I read a little bit about the `save` in the code and couldn't get the full picture. Can you explain a little bit on this (i.e. when and why use `save()`? Thanks!

And please add this to the docstring for both `emit()`, one for `NotificationSubscription` and the other for `NotificationType`. Note: I was able to understand the other args of `emit()` from your docstring 👍.